CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,75 @@
+Contributor Code of Conduct
+===========================
+
+Our Pledge
+----------
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers of CNERG pledge to making participation in our project and
```

Comment:
@gonuke - Do you think this should say the "maintainers of CNERG" or "maintainers of Svalinn"?

Reply:
maybe "the Svalinn software suite, including DAGMC,"